api/envoy/extensions/filters/http/ext_proc/v3alpha/processing_mode.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.ext_proc.v3alpha;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.ext_proc.v3alpha";
+option java_outer_classname = "ProcessingModeProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: External Processing Filter]
+// External Processing Filter Processing Mode
+// [#extension: envoy.filters.http.ext_proc]
+
+// This configuration describes how an external processing filter processes the HTTP
+// request or response, including how requests and responses are
+// sent to the remote server.
+
+// [#next-free-field: 8]
+message ProcessingMode {
+  // Control how headers and trailers are handled
+  enum HeaderSendMode {
+    // The default HeaderSendMode depends on which part of the message is being
+    // processed. By default, request and response headers are sent, while
```

Comment:
```suggestion
    // processed. By default, request and response headers are sent,
```

Stepping back this comment is confusing to me. Why is body referenced? I assume we are just talking about headers and trailers here and trailers are default skipped?